include/aikido/robot/ConcreteRobot.hpp

```diff
@@ -102,6 +102,20 @@ class ConcreteRobot : public Robot
       const dart::dynamics::MetaSkeletonPtr& metaSkeleton,
       const constraint::dart::CollisionFreePtr& collisionFree) const override;
 
+  // Get a smoothing post postprocessor that respects velocity and acceleration
+  // limits, as well as the passed constraint.
+  /// \param[in] metaSkeleton The MetaSkeleton whose limits must be respected.
+  std::shared_ptr<aikido::planner::TrajectoryPostProcessor>
+  getTrajectoryPostProcessor(
```

Comment:
I'm fine with this for now, but in light of the discussion from #443 it seems that maybe it doesn't make sense to have this here? I think we should probably set up the postprocessor in the concrete robot constructor, similar to how we set up the metaplanner.